### DISCUSSION TITLE
multipart는 POST로만 보내야하는 것에 대한 논의

### DISCUSSION THREAD

**Discussion by june0216** [EFUB-LakkuLakku/LakkuLakku-Back#104] [Ideas]
PUT은 특정 리소스를 갱신하는 역할을 하는데 multipart로 보내면 한번에 여러 리소스를 처리하므로 이미지 같은 경우를 PUT으로 처리하려면 이미지등에 대한 리소스 URI에 별도의 PUT 요청을 보내서 갱신하고 일반적인 폼은 따로 처리해야 한다. 라꾸라꾸 profileUpdate에서 한 URI로 PUT을 보내서 여러 리소스(회원 정보 + 이미지)를 한꺼번에 처리하려고 했으므로 PUT이 적합치 않으므로 POST로 개선해야할 것 같습니다. 

관련 링크 = https://blog.outsider.ne.kr/1001

**Comment by nitronium102** [comment]
+ multipart는 GET/POST만 지원됩니다
